### PR TITLE
[CMS] Validation messages missing from datepicker form elements

### PR DESCRIPTION
**Pull request by vasilii-b**
### Description (*)
This PR adds both frontend and backend validation for the "Custom Theme" From-To date range.
The original PR #17520 was created to be merged to `2.2-develop` and is inactive for a while. Moreover, there is not added the backend validation.

xnip2019-01-30_11-26-45

### Fixed Issues (if relevant)
1. magento/magento2#3795 - Validation messages missing from datepicker form elements

### Manual testing scenarios (*)
Provided in task description

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @vasilii-b. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/code/Magento/Cms/Model/ResourceModel/Page.php [hunk 1]
```diff
@@ -265,6 +273,28 @@ protected function isValidPageIdentifier(AbstractModel $object)
...
     }
 
+    /**
+     * Chech whether page custom theme from date is
+     * lower than custom theme to date
+     *
+     * @param AbstractModel $object
+     * @return bool
+     */
+    private function isValidPageCustomThemeDateRange(AbstractModel $object)
```
Comment (rogyar) [inline app/code/Magento/Cms/Model/ResourceModel/Page.php]: One more thing, should we clarify the type hinting here by using page model instead fo AbstractModel?

Comment (vasilii-b) [reply app/code/Magento/Cms/Model/ResourceModel/Page.php]: Hi @rogyar, I've changed it to the CMS Page. Thank you!

**Comment by rogyar** [comment]
@vasilii-b could you check the failing tests, please? Thank you

**Comment by sidolov** [comment]
@vasilii-b , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Comment by VladimirZaets** [comment]
@vasilii-b, I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for collaboration

**Pull request closed without merging**